src/components/ExportCSV/ExportCSV.jsx

```diff
@@ -0,0 +1,326 @@
+        </div>
+        <div className="right-column">
+          <div className="two-text-header">
+            <p style={{ fontSize: '36px', fontWeight: '700' }}>Export CSV</p>
+            <p style={{ fontSize: '26px' }}>30 boxes</p>
+          </div>
+          <div className="section">
+            <p className="option-header">Sort By</p>
+            <Select fontSize="20px" value={sort} onChange={e => setSort(e.target.value)}>
+              <option value="ascend-box-num">Ascending Box Number</option>
+              <option value="descend-box-num">Descending Box Number</option>
+              <option value="chronologic">Chronologically</option>
+              <option value="ascend-zip-code">Descending Zip Code</option>
+              <option value="descend-zip-code">Descending Zip Code</option>
+            </Select>
+          </div>
+          <div className="section">
+            <p className="option-header">Filter Options</p>
+            <div className="filters">
+              <div className="filter-options">
+                <p className="filter-names">Boxes</p>
+                <div className="drop-option">
+                  <Select
+                    fontSize="20px"
+                    width="300px"
+                    value={boxes}
+                    onChange={e => setBoxes(e.target.value)}
+                  >
+                    <option value="all">All</option>
+                    <option value="custom">Custom</option>
+                  </Select>
+                  {boxes === 'custom' && (
+                    <div>
+                      <Input
+                        value={boxesCustom}
+                        className="custom-text"
+                        placeholder="e.g. 1-9, 6, 12"
+                        isInvalid={checkRange(boxesCustom)}
+                        onChange={e => setBoxesCustom(e.target.value)}
+                      />
+                      {checkRange(boxesCustom) && (
+                        <p className="error-message">Invalid page range, e.g. 1-9, 6, 12</p>
+                      )}
+                    </div>
+                  )}
+                </div>
+              </div>
+              <div style={{ display: 'flex', flexDirection: 'column' }}>
+                <div className="filter-options">
+                  <p className="filter-names">Date</p>
+                  <div className="drop-option">
+                    <Select
+                      fontSize="20px"
+                      width="300px"
+                      value={date}
+                      onChange={e => setDate(e.target.value)}
+                    >
+                      <option value="all">All</option>
+                      <option value="single">Single Day</option>
+                      <option value="range">Range</option>
+                    </Select>
+                    {date === 'single' && (
+                      <div>
+                        <Input
```

Comment:
Can we change this input to use React datepicker instead? This way you don't need to validate the date and the user can just select a date instead of typing.
https://yarnpkg.com/package/react-datepicker